### PR TITLE
Various bug fixes

### PR DESCRIPTION
**Pull request by jacquarg**
* avoid replication without initialization (try-fix overconsumption bug)
* simplification in wifi detection
* fix error in photo backup with ':' char in path
* continue on error in photo backup
* remove url field from cozy contact.
* (fix crashes in plugins).

### CODE REVIEW

**Comment by frankrousseau** [comment]
@jacquarg can you rebase it with @kelukelu changes?

**Comment by jacquarg** [comment]
I just broke sliding feature on folder lines. But the previous behavior was bad too...

**Comment by frankrousseau** [comment]
@aenario can you review it?

**Comment by frankrousseau** [comment]
Can you rebase this one @jacquarg?

File: www-src/app/application.coffee [hunk 1]
```diff
@@ -49,10 +50,22 @@ module.exports =
...
                 Backbone.history.start()
 
+                DeviceStatus.initialize()
+
                 if config.remote
-                    app.regularStart()
+                    unless @replicator.config.has('checkpointed')
+                        log.info 'Launch first replication again.'
+                        app.replicator.initialReplication (err) ->
+                        if err
```
Comment (aenario) [inline www-src/app/application.coffee]: wrong indent

File: www-src/app/application.coffee [hunk 2]
```diff
@@ -82,13 +95,13 @@ module.exports =
             app.replicator.stopRealtime()
 
         , false
-        document.addEventListener 'offline', ->
-            DeviceStatus = require './lib/device_status'
-            DeviceStatus.update()
-        , false
+        # document.addEventListener 'offline', ->
```
Comment (aenario) [inline www-src/app/application.coffee]: remove lines instead of commenting

File: www-src/app/views/folder.coffee [hunk 3]
```diff
@@ -54,6 +52,14 @@ module.exports = class FolderView extends CollectionView
                     gesture.distance = gesture.deltaX
                     gesture.velocityY = 0
 
+                # else
```
Comment (aenario) [inline www-src/app/views/folder.coffee]: Remove this lines

**Comment by aenario** [comment]
There is also a conflict situation with whith https://github.com/cozy/cozy-mobile/pull/75.

@lemelon , could you wait for this PR to be merged before rebasing yours and include new translations into transifex / your PR

**Comment by jacquarg** [comment]
@aenario just clean it. Thanks for your careful review !

**Pull request merged**